Sources/BraintreePayPalMessaging/BTPayPalMessagingAnalytics.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+enum BTPayPalMessagingAnalytics {
+
+    static let started = "paypal-messaging:create-view:started"
+    static let failed = "paypal-messaging:create-view:failed"
+    static let succeeded = "paypal-messaging:create-view:succeeded"
+}
```

Comment:
The PayPal Messages team collects their own analytics so we can decide what we want analytics for. I opted to keep things super simple for now, but if we feel that other events are important to capture we can add them. We can additionally add more events in the future if we decide that we are not getting enough data from these.

Reply:
Can we add these to our events excel doc + description?

Reply:
Maybe with a note that they're "to be released"?

Reply:
Added!